jupyterlab_git/handlers.py

```diff
@@ -822,6 +822,10 @@ async def post(self, path: str = ""):
             self.set_status(500)
         self.finish(json.dumps(body))
 
+    @tornado.web.authenticated
+    def check_ignore(self, path):
+        self.git.check_ignore(self, path)
```

Comment:
Here you want this method to be executed on an `GET` request. So the method should be named `get`. Then you will obtain the repository path as `path` input argument and you could for example get the relative path to check through a query argument like this:

```suggestion
    async def get(self, path):
        path_to_check = self.get_query_argument("path", "")
        body = await self.git.check_ignore(self, self.url2localpath(path), path_to_check)

        if body["code"] != 0:
            self.set_status(500)
        self.finish(json.dumps(body))
```